tests/test_pacing.py

```diff
@@ -55,6 +66,18 @@ def test_interpolations(self, mocked_narrator) -> None:
         completed_line = mocked_narrator.stating_a_beat.call_args_list[0][0][1]
         assert completed_line == f"The {test_weapon} in the {test_room}!"
 
+    def test_beat_logging_none(self, Tester, caplog):
+        caplog.set_level(logging.INFO)
+        See(NonesyQuestion(), IsEqualTo(None)).perform_as(Tester)
+
+        assert [r.msg for r in caplog.records] == [
+            "Tester sees if nonesyQuestion is equal to None.",
+            "    Tester examines NonesyQuestion",
+            "        => None",
+            "    ... hoping it's equal to None.",
+            "        => <None>",
+        ]
+
```

Comment:
I think we need one more test of `beat` logging a not-Answerable method.

Also, if someone does something like this:
```python
class CornerCase:
    def answered_by(self, the_actor):
        ...
        
    @beat("Blah!")
    def do_not_log_me_man(self) -> None:
        # i just do something but don't want my return value to be logged
```
the code as-is will log `=> <None>` for a call to `.do_not_log_me_man()`, right? We might need to go a step further and compare the method name *and* the class against the protocol.

Reply:
So we are back to the dilemma; how do we know if a function SHOULD return a None object that we want logged?

We made exceptions for `answered_by` and `resolve` by checking if we are currently attached to a class which conforms to the protocols `Answerable` and `Resolvable`.  Should we take it a step further and see if we are in methods with those specific names?  That rules out any functions that could legitimately return None we want logged as well.

I thought about trying to use the annotation of the function.  If it annotates with ONLY `-> None` we skip logging, but if it's anything else we allow it to log?

Reply:
Say a user wanted to use beat on any function that returns.  Should THOSE get logged?

```python
@beat("does thing")
def foo() -> Any:
    return x
```
If so, limiting to protocols doesn't work here.

Reply:
>  If it annotates with ONLY -> None we skip logging, but if it's anything else we allow it to log?"

Oh, i kinda like that idea. That would skip us having to maintain a list of protocols which can/can't log and instead just use the annotation to see if we should be logging—yes, i do like that!

Reply:
I'm glad to hear you like it, cuz I can't shake the feeling I'm doing something very wrong.

Reply:
Is it that annotations aren't supposed to have runtime implications? Well we've crossed the Rubicon on that one 'cause we're already using Pydantic. :P

Reply:
Does this mean I should take this out of draft and officially submit it?

Reply:
Yes please! Let's get this reviewed and merged in, i like this approach.